service/src/registry.ts

```diff
@@ -115,6 +115,13 @@ export class Registry {
    * @method            module:azure-iothub.Registry#update
    * @description       Updates an existing device identity on an IoT hub with
    *                    the given device information.
+   *                    The `deviceInfo` parameter must include all properties
+   *                    that are updatable.  For example, if updating the `status`
+   *                    property from `disabled` to `enabled`, the `deviceInfo`
+   *                    object should also have the statusReason, authentication
+   *                    (and its sub-properties), capabilities, and deviceScope
+   *                    set to the devices current values or those properties will
+   *                    be reset.
```

Comment:
Do you think it's self explanatory that the recommended developer workflow be _Get the device identity, then copy all of the fields into the update API_? Or should you explain that workflow?

Reply:
done.